Client/src/App.jsx

```diff
@@ -27,12 +27,12 @@ export default function App() {
     <Router>
       {/* <UserContext.Provider value={{ userData, userLoading, userError }}> */}
       <Routes>
-        <Route path = "/" element = {<SignUp />} />
-        <Route path = "/Login" element = {<Login />} />
-
-        <Route element={ <PrivateRoutes/>}>
+        {/* <Route path = "/" element = {<SignUp />} />
```

Comment:
Please remove the commenting out of the other routes here.  That's a good way to get around having to sign in during development, but we want the sign in page to be the first page loaded generally.